corehq/form_processor/models/cases.py

```diff
@@ -47,6 +49,41 @@ def get_case(self, case_id, domain=None):
         except CommCareCase.DoesNotExist:
             raise CaseNotFound(case_id)
 
+    def get_cases(self, case_ids, domain=None, ordered=False, prefetched_indices=None):
+        """
+        :param case_ids: List of case IDs to fetch
+        :param domain: Currently unused, may be enforced in the future.
```

Comment:
do you have a plan for this?

Reply:
Not at the moment. It would be easy enough to implement, but I haven't taken time yet to carefully consider possible negative outcomes. Also didn't want to change this one as part of the larger _moving code_ refactor so as not to introduce too many variables.

I kept the argument here because I wanted to preserve the domain associated with `CaseAccessors` so it would not be necessary to change every `CommCareCase.objects.get_cases(...)` call when this is done in the future.